crates/rocksdb/src/lib.rs

```diff
@@ -353,6 +353,55 @@ impl RocksDb {
         self.manager.async_spawn(task).await?
     }
 
+    #[tracing::instrument(skip_all, fields(db = %self.name))]
+    pub async fn import_cf(
+        &self,
+        name: CfName,
+        opts: &RocksDbOptions,
+        metadata: ExportImportFilesMetaData,
+    ) -> Result<(), RocksError> {
+        // todo(pavel): perf guard
+        let default_cf_options = self.manager.default_cf_options(opts);
+        let cf_patterns = self.cf_patterns.clone();
+        let db = self.db.clone();
+        let task = StorageTask::default()
+            .kind(StorageTaskKind::ImportColumnFamily)
+            .op(move || db.import_cf(name, default_cf_options, cf_patterns, metadata))
```

Comment:
Those tasks (+export) are good candidates to have low priority (`.priority(Priority::Low)`)